harmony-kotlin/src/commonMain/kotlin/com.harmony.kotlin/data/datasource/memory/InMemoryDataSource.kt

```diff
@@ -13,16 +12,14 @@ import com.harmony.kotlin.error.notSupportedQuery
 
 class InMemoryDataSource<V> : GetDataSource<V>, PutDataSource<V>, DeleteDataSource {
```

Comment:
With the new memory model IsolateState shouldn't be necessary. 

Removed the fix and still passing the test

Reply:
If it passes the tests, all ok. Another dependency was removed.